doc/en/components/grids/_shared/search.md

```diff
@@ -75,9 +76,65 @@ Let's start by creating our grid and binding it to our data. We will also add so
     <IgrColumn field="MarketNotion" dataType="number" sortable="true"></IgrColumn>  
 </{ComponentSelector}>
 ```
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
```

Comment:
Update code snippets to be up to date with the latest changes to the [Blazor](https://github.com/IgniteUI/igniteui-blazor-examples/pull/708), [React](https://github.com/IgniteUI/igniteui-react-examples/pull/548) and [Web Components](https://github.com/IgniteUI/igniteui-wc-examples/pull/736) samples.